decidim-elections/app/cells/decidim/votings/content_blocks/metrics_cell.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module ContentBlocks
+      class MetricsCell < Decidim::ContentBlocks::ParticipatorySpaceMetricsCell
+        def metrics
+          nil
+
+          # @metrics ||= VotingsMetricChartsPresenter.new(participatory_process: resource)
```

Comment:
@entantoencuanto should be uncommented?

Reply:
It was like that previously, this file has been only moved. If anything, the line should be removed.

Reply:
Added a comment